go.mod

```diff
@@ -5,6 +5,7 @@ go 1.18
 require (
 	github.com/go-openapi/strfmt v0.19.3
 	github.com/go-openapi/swag v0.19.5
+	github.com/go-test/deep v1.0.3
```

Comment:
I see that the go-test/deep is being used in non test files, can this be avoided  ?

Reply:
Sure, thanks for pointing this out, I will change it